multichain-testing/package.json

```diff
@@ -91,29 +91,5 @@
     "@agoric/zoe": "portal:../packages/zoe",
     "@agoric/zone": "portal:../packages/zone"
   },
-  "eslintConfig": {
```

Comment:
please separate `multichain-testing` changes to another PR. It's complicating this migration of agoric-sdk root project. Let's get that landed.

Reply:
Without these changes, linting will fail for multichain-testing as it'll pick up the root config and try to use that

Reply:
Hmm, I just tried it and it seemed to work.
```
cd multichain-testing
git checkout master .
yarn install
yarn lint
```
Do you get a different result?

Reply:
btw, assuming we can punt updating `multichain-testing`, this PR should no longer be marked as _closing_ 10642

Reply:
This _does_ work locally. I'll remove the changes and observe CI. It _was_ picking up the new config in CI, but maybe I just observed the wrong thing and chased the wrong path

Reply:
Going back to [this](https://github.com/Agoric/agoric-sdk/pull/10846#discussion_r1924433805) comment, when you ran `git checkout master .`, I don't think it removed the `eslint.config.mjs` file, so when you ran `yarn lint`, it was still using that file

Reply:
Very confusing when CI has a different result for `yarn lint` than the local checkout does.  I tried diagnosing why [the linked job](https://github.com/Agoric/agoric-sdk/actions/runs/12897014411/job/35961257562?pr=10846) failed but I couldn't determine what state of this branch it was using and what changes it had to multichain-testing.

So I made a new branch, https://github.com/Agoric/agoric-sdk/tree/ta/eslint-v9, which has the steps I pasted above. It's passing in https://github.com/Agoric/agoric-sdk/actions/runs/12898290669/job/35965066953

Maybe the failing job did not have the master state of `multichain-testing`?

Reply:
Even when you reverted the change in the `eslint-v9` branch, the new `eslint.config.mjs` file is still [present](https://github.com/Agoric/agoric-sdk/blob/27bdd7691ba701c0d766800814a44a4e7d2d91f3/multichain-testing/eslint.config.mjs) in the latest version of the branch

Reply:
Ah. My branch doesn't actually match what I was asking for. And upon closer inspection it looks like it's green because in master the linter is broken in this dir.

So thanks for fixing it!